inst/shiny/DiagnosticsExplorer/server.R

```diff
@@ -542,6 +550,118 @@ shiny::shinyServer(function(input, output, session) {
     return(table)
   })
   
+  output$temporalCharacterizationTable <- DT::renderDataTable({
+    data <- temporalCovariateValue %>% 
+            dplyr::filter(.data$cohortId == cohortId(),
+                          .data$databaseId == input$databases,
+                          .data$timeId == timeId())
+    data$cohortId <- NULL
+    databaseIds <- unique(data$databaseId)
+    
+    if (input$charTypeTemporal == "Pretty") {
```

Comment:
You should ask Patrick what he thinks, but I don't think a 'Pretty' view makes sense for temporal covariates, since we're not trying to get an overall picture of the cohort, but are looking for specific codes that are behaving differently than expected. Just a raw view, sorted by prevalence, would be most helpful.